ansible/roles/kafka/tasks/deploy.yml

```diff
@@ -60,17 +60,3 @@
     settings: "{{ kafka.topics.health }}"
```

Comment:
What about the health topic? Why not have them all try to create it and the first one wins?

Reply:
I thought about that.  Same for the cacheInvalidation topic.  The controllers could just race to create them on startup and one of them will win.  If we think it is a good idea, I can do a followup PR to get all the topic logic out of ansible.

Should the followup PR wait on the pureconfig PR?  If not, I'll have to plumb more kafka-specific flags through the WhiskConfig.